packages/commands/src/index.ts

```diff
@@ -613,15 +635,19 @@ export class CommandRegistry {
    */
   private _executeKeyBinding(binding: CommandRegistry.IKeyBinding): void {
     let { command, args } = binding;
-    if (!this.hasCommand(command) || !this.isEnabled(command, args)) {
+    let newArgs: ReadonlyPartialJSONObject = {
+      _luminoEvent: { type: 'keybinding', keys: binding.keys },
+      ...args
+    };
```

Comment:
Not sure about modifying the args of the command.
Can you explain what is the objective ?

Reply:
Part of a manual re base I did, The intended objective is covered here:
https://github.com/jupyterlab/lumino/commit/3aaa56f7dc4a3ddc48f4fa985fcbedcfd716a947

Reply:
OK, that shouldn't be in this PR. Can you fix it ?